### PR TITLE
bump to rails 4 and ruby 2.1.6

### PR DESCRIPTION
**Pull request by zassmin**
@jdoconnor and @azach fulfills https://lumoslabs.atlassian.net/browse/WE-296

### CODE REVIEW

**Comment by azach** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ rvm:
   - 1.9.3
   - 1.9.2
   - 2.0.0
+  - 2.1.6
```
Comment (zassmin) [inline .travis.yml]: Thinking about removing all the other ruby versions.

Comment (azach) [inline .travis.yml]: I think it's fine to support until it becomes a burden

File: spec/dummy/config/application.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 require "active_record/railtie"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-require "active_resource/railtie"
+# require "active_resource/railtie"
```
Comment (jdoconnor) [inline spec/dummy/config/application.rb]: please remove commented code

File: .travis.yml (hunk 1 above)
Comment (jdoconnor) [inline .travis.yml]: I don't think anyone uses this, so I don't see an issue with removing. I'm pushing for removing this gem altogether from our apps if possible.

**Pull request merged**